src/react-apps/applications/ux-editor/package.json

```diff
@@ -28,9 +28,8 @@
     "dot-object": "^1.7.0",
     "immutability-helper": "2.7.0",
     "lodash.isequal": "^4.5.0",
-    "react": "16.6.3",
-    "react-beautiful-dnd": "^10.0.2",
-    "react-dom": "16.6.3",
+    "react": "16.6.1",
+    "react-dom": "16.6.1",
```

Comment:
any specific reason the version in downgraded?

Reply:
Nop, just my package.json being outdated.